openeo/files.js

```diff
@@ -88,6 +91,12 @@ var Files = {
 		req.on('end', () => {
 			stream.end();
 			res.send(200);
+			const payload = {
+				user_id: req.user._id,
+				path: p.replace('storage/user_files/'+req.user._id+'/', ''),
+				action: 'created'
+			};
+			this.subscriptions.broadcast('openeo.files', {}, payload); // TODO: params shouldn't be empty object but the payload, so that clients can subscribe to file-specific messages etc.
```

Comment:
Hmm... we should discuss whether this should be a broadcast. Should file subscriptions be user-specific for now?

Reply:
I named the function `broadcast`, however it doesn't do a "_full_ broadcast" to EVERY subscriber, but instead the `broadcast` function checks who is subscribed to this topic with these parameters and notifies the appropriate subscribers (so it's more a "broadcast to those who tuned in to this type of message"). I like this name for this function, but feel free to suggest alternatives.

Reply:
"Multicast" is a very rare word that isn't immediately clear - I'll go for `publish` since it's a publish/subscribe mechanism